docs/fundamentals/executor/hub/use-hub-executor.md

```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
 
 
 The source code of the Executor will be stored at `~/.jina/hub-packages`.
+
+### list the local location of executors
+
+```bash
+jina hub list
```

Comment:
can u please add an output example?

Reply:
We'll record a command line video about it